app/src/main/java/space/taran/arknavigator/ui/adapter/TagsSelectorAdapter.kt

```diff
@@ -25,10 +27,16 @@ class TagsSelectorAdapter(
         drawAvailableTags()
         drawClearBtn()
         drawUnavailableTags()
+
+        presenter.availableTagsForDisplay.forEach { tag ->
```

Comment:
I think we don't need to do it every time we re-draw tags. Can be done once during initialization. User also can change tags after he jumped to Resources screen.